docs/docs/adrs/adr-019-fault-resolutions.md

```diff
@@ -0,0 +1,131 @@
+---
+sidebar_position: 20
+title: Fault Resolutions
+---
+# ADR 019: Fault Resolutions
```

Comment:
Should fault resolutions be optional (?) and a consumer chain can choose whether to use them or not when issuing a `MsgCreateConsumer` or a `MsgUpdateConsumer`. What's the relation with [docs: ADR for Customizable Slashing and Jailing](https://github.com/cosmos/interchain-security/pull/2146/files)?